### PR TITLE
fix lint rules

### PR DESCRIPTION
**Pull request by xuke444**
Signed-off-by: Ke Xu <xuke@microsoft.com>

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: libs/core-ui/.eslintrc.json [hunk 1]
```diff
@@ -16,9 +16,7 @@
     },
     {
       "files": ["*.ts", "*.tsx"],
-      "rules": {
-        "max-lines": "error"
```
Comment (tongyu-microsoft) [inline libs/core-ui/.eslintrc.json:20]: why do we remove this rule for core-ui? I didn't see we set global max-lines rule in this pr

File: libs/e2e/src/util/BoxChart.ts [hunk 2]
```diff
@@ -59,10 +59,13 @@ export class BoxChart extends Chart<IBox> {
       : { ...boxCoordinate };
   };
 
-  private readonly getBoxCoordinate = (
+  private readonly getD = (
```
Comment (tongyu-microsoft) [inline libs/e2e/src/util/BoxChart.ts:62]: for my learning, what is this `D`?

**Review by tongyu-microsoft — APPROVED** [review]

File: libs/core-ui/.eslintrc.json (hunk 1 above)
Comment (xuke444) [reply libs/core-ui/.eslintrc.json:20]: It is in .eslintrc.custom.eslintrc as a warn.
I was planning to fix all in this pr, but seems it is too big already, I will fix all max line issues in following pr.

File: libs/e2e/src/util/BoxChart.ts (hunk 2 above)
Comment (xuke444) [reply libs/e2e/src/util/BoxChart.ts:62]: The d value of svg element.
This is a extra function added to address cyclone complexity.
Seems git hub did not recognize it correctly.
getBoxCoordinate is still below

**Pull request merged**